modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/senna/events/EventEmitter.js

```diff
@@ -0,0 +1,478 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import Disposable from '../Disposable';
+import utils from '../utils/utils';
+import EventHandle from './EventHandle';
+
+const singleArray_ = [0];
```

Comment:
We should drop the trailing underscore from this module-private variable. The trailing underscore on the "private" member properties on the class is also not great, but this one seems totally pointless because it is not even visible outside the scope of the module.

Reply:
Ugh... I thought I did most, but maybe lost those in the commit mayhem :(

Reply:
https://github.com/liferay-frontend/liferay-portal/pull/489#issuecomment-725325366